src/Microsoft.AspNet.Diagnostics.Elm/ElmCaptureMiddleware.cs

```diff
@@ -28,19 +29,15 @@ public ElmCaptureMiddleware(RequestDelegate next, ILoggerFactory factory, IOptio
 
         public async Task Invoke(HttpContext context)
         {
-            var requestId = Guid.NewGuid();
-            using (_logger.BeginScope(string.Format("request {0}", requestId)))
+            var p = ElmScope.Current;
```

Comment:
what's the point of this line? It looks like a no-op. It doesn't look like it did anything before either.

In fact, it looks like the next line would easily null ref.